import/dproto/serialize.d

```diff
@@ -460,19 +467,19 @@ void writeProto(string T, R)(ref R r, BuffType!T src)
 }
 
 /// Ditto
-void writeProto(string T, R)(ref R r, const BuffType!T src)
+void writeProto(string T, R)(ref R r, const BuffType!T src) @trusted
 	if(isProtoOutputRange!R &&
 	  (T.msgType == "double".msgType || T.msgType == "float".msgType))
 {
 	r.put(src.nativeToLittleEndian!(BuffType!T)[]);
 }
 
 /// Ditto
-void writeProto(string T, R)(ref R r, const BuffType!T src)
+void writeProto(string T, R)(ref R r, const BuffType!T src) @trusted
 	if(isProtoOutputRange!R && T.msgType == "string".msgType)
 {
 	toVarint(r, src.length);
-	r.put(cast(ubyte[])src);
+	r.put(cast(const ubyte[]) src);
```

Comment:
We don't expect `put` to make changes to `src`. Do we? Casting something const to something non-const feels itchy and is prohibited in `@safe` code.